### PR TITLE
Removed dependence on views

### PR DESCRIPTION
**Pull request by libern**
Hi, I have make it works even the app is not ready or view is not available via static way.

### CODE REVIEW

File: src/Displayers/StaticDisplayer.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+     * @var string
+     */
+    protected $view;
+
+    /**
+     * Create a new static displayer instance.
+     */
+    public function __construct()
+    {
+        $this->view = file_get_contents(__DIR__.'/../../views/plain.html');
```
Comment (GrahamCampbell) [inline src/Displayers/StaticDisplayer.php]: This means the view contents is nolonger configurable. The view files are normally overridable by "publishing" them to the app.

**Comment by GrahamCampbell** [comment]
To be honest, it might be worth just replacing the plain displayer with this. It's pretty nice.

File: src/Displayers/StaticDisplayer.php (hunk 1 above)
Comment (libern) [inline src/Displayers/StaticDisplayer.php]: I think there should be a way to get the path to the published directory. As you didn't mention the view files are configureable through publishing them in the document, so I didn't do that part.

Comment (GrahamCampbell) [inline src/Displayers/StaticDisplayer.php]: If we replace the plain handler with this, we might as well ditch the configuration bit.

**Comment by GrahamCampbell** [comment]
Manually merged and made a few changes. Thank you for this. :)

**Pull request closed without merging**